package-drone/src/main/java/de/dentrassi/pm/jenkins/UploaderV3.java

```diff
@@ -104,6 +104,7 @@ public void upload ( final File file, final String filename ) throws IOException
         }
         catch ( final IOException e )
         {
+            e.printStackTrace ( this.listener.error ( "Failed to create upload archive" ) );
```

Comment:
Sorry, I overlooked that. You actually are sending the output to the listener.

My mistake!